diff --git a/dist/utils/async_caller.js b/dist/utils/async_caller.js
index c1e1dff60746ec34b4fd5baa805f0e86d034d46a..c60354c033105a8888499d0d7da33955c9b5e86f 100644
--- a/dist/utils/async_caller.js
+++ b/dist/utils/async_caller.js
@@ -1,5 +1,5 @@
-import pRetry$1 from "../node_modules/.pnpm/p-retry@7.1.1/node_modules/p-retry/index.js";
-import PQueue from "../node_modules/.pnpm/p-queue@9.1.0/node_modules/p-queue/dist/index.js";
+import pRetry$1 from "p-retry";
+import PQueue from "p-queue";
 import { _getFetchImplementation } from "../singletons/fetch.js";
 //#region src/utils/async_caller.ts
 /**
